File: rb/BeginnersSelection/02_PlacingMarbles.rb
# https://atcoder.jp/contests/abs/tasks/abc081_a

class Object
  def let(f)
    f.call self
  end
end

gets
  .chomp
  .split("")
  .map(&:to_i)
  .sum
  .let(method(:puts))

# https://atcoder.jp/contests/abs/submissions/21508228

File: jl/BeginnersSelection/02_PlacingMarbles.jl
# https://atcoder.jp/contests/abs/tasks/abc081_a

(
  readline()
  |> s -> [c for c = s]
  |> a -> sum(x -> parse.(Int, x), a)
  |> println
)

# https://atcoder.jp/contests/abs/submissions/22542702
